drivers/bridge/bridge.go

```diff
@@ -647,7 +646,7 @@ func (d *driver) createNetwork(config *networkConfiguration) error {
 		id:         config.ID,
 		endpoints:  make(map[string]*bridgeEndpoint),
 		config:     config,
-		portMapper: portmapper.New(d.config.UserlandProxyPath),
+		portMapper: portmapper.New(portmapper.WithUserlandProxy(d.config.EnableUserlandProxy, d.config.UserlandProxyPath)),
```

Comment:
Also one reason I did it this way is because otherwise the function API is a bit uglier because you have to first group the options you want and then pass them in as a slice.